engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/validation/ZeebeExpressionValidator.java

```diff
@@ -99,6 +103,19 @@ private static void verifyPath(
     }
   }
 
+  public static boolean isListOfCsv(final Expression staticExp) {
+    final String value = staticExp.getExpression();
+    if (value.isEmpty()) {
+      return true;
+    }
+    final var values = value.split(",");
+    if (values.length < StringUtils.countMatches(value, ",") + 1) {
+      // one of the values was an empty string, e.g. [a,,c]
```

Comment:
:wrench: Fix comment.

```suggestion
      // one of the values was an empty string, e.g. 'a,,c'
```